### ISSUE TITLE
Integraing Algorithm with Different Benchmarks

### ISSUE THREAD

**Issue by vaggelisT** [facebookresearch/modem#5]
Hello, 

I am interested in integrating this RL algorithm with different benchmark environments. Could you please guide me on whether this is feasible? If yes, what specific changes and additions would I need to implement to achieve this integration successfully? 

Thank you.

**Comment by nicklashansen** [comment]
Yes, this is certainly possible and should not be too much work! You would need to add your environment to the `tasks` directory here: https://github.com/facebookresearch/modem/tree/main/tasks and collect a few demonstrations for use with MoDem. To collect demonstrations you can either use motion planning, a scripted policy, human teleoperation, or simply train an RL policy with e.g. dense rewards / privileged information.
